arch/x86/kvm/pkvm/pgtable.h

```diff
@@ -102,4 +143,28 @@ int pkvm_pgtable_init(struct pkvm_pgtable *pgt,
 		      struct pkvm_pgtable_mm_ops mm_ops,
 		      struct pkvm_pgtable_ops pgt_ops);
 
+/*
+ * pkvm_pgtable_walk() - Walk a pkvm page table
+ *
+ * @pgt:	The page table to walk.
+ * @vaddr:	The virtual address for the start of the walk.
+ * @size:	The walk range size.
+ * @walker:	The callback descriptor which will be executed
+ *		during the walk.
+ *
+ * The vaddr will be aligned down with PAGE_SIZE, and the size will
```

Comment:
Use @ whenever arguments are referenced, i.e. `@vaddr`, `@size` etc (to be kerneldoc-friendly)?